config/environments/test.rb

```diff
@@ -40,6 +40,7 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
check this, or remove it

Reply:
done

Reply:
what have you done, You haven't removed this line, neither did you explain why it is there in test.rb ?